trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -209,51 +216,87 @@ const mapPaymentToPaymentDetailsPaymentViewModel = (
   },
   reference: payment.reference,
   dateReceived: {
-    formattedDateReceived: format(new Date(payment.dateReceived), 'd MMM yyyy'),
+    formattedDateReceived: format(new Date(payment.dateReceived), DATE_FORMAT.DAY_SHORT_MONTH_YEAR),
     dataSortValue: dateReceivedDataSortValue,
   },
 });
 
 /**
- * Returns the payments sorted by date received
- * @param payments - The payments to sort
- * @returns Sorted payments
+ * Gets the formatted reconciled by user
+ * @param reconciledByUser - The reconciled by user
+ * @returns The formatted reconciled by user
+ */
+export const getFormattedReconciledByUser = (reconciledByUser: { firstName: string; lastName: string } | undefined): string =>
+  reconciledByUser ? `${reconciledByUser.firstName} ${reconciledByUser.lastName}` : '-';
+
+/**
+ * Formats the date reconciled
+ *
+ * The "aaaaa'm'" selection is so we can format the date to use
```

Comment:
Can we use the name of the constant here instead?

Reply:
I've moved the in depth explanation to the constant and removed the repeat of part of the constant